lms/static/sass/views/_login-register.scss

```diff
@@ -371,6 +369,7 @@ $sm-btn-linkedin: #0077b5;
             height: 34px;
             line-height: 34px;
```

Comment:
Re: "Please remove and find a better way to space the icons in this case."

I converted this to a sass variable, which is at least a bit of an improvement. I tried finding different ways of spacing this icon, but found that due to the nature of the design (a solid overflowing rectangle left-aligned inside a `button` element), the only options seem to be to change the visual appearance significantly or to restructure the HTML so that the CSS can be re-written to use `display: table-row` and `display: table-cell` or some other quite different approach. Since those buttons and their CSS were pre-existing and I didn't really modify them other than adding a default background color (for buttons that are neither Google nor Facebook nor LinkedIn) and making a few CSS fixes you asked for, I hope we can consider such a fix out of scope of this PR.